extensions/interactions/NumericExpressionInput/directives/numeric-expression-input-validation.service.ts

```diff
@@ -35,6 +36,8 @@ import { AppConstants } from 'app.constants';
   providedIn: 'root'
 })
 export class NumericExpressionInputValidationService {
+  private supportedFunctionNames = AppConstants.SUPPORTED_FUNCTION_NAMES;
```

Comment:
Why is this needed?

Reply:
This is used to show the list of supported functions via the validation error message.